iac/terraform/aws/web-client/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "cloudfront_domain_name" {
```

Comment:
Let's add a bit of context to the output variable name.

Reply:
Okay.